packages/components/src/Dropzone/sub-components/DropzoneError/DropzoneError.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { ErrorCode, FileError, FileRejection } from "react-dropzone";
+import { InlineNotification } from "@kaizen/notification";
+import { getKaizenDropzoneError, prettifyErrorCode } from "../../utils";
+
+export const DropzoneError = ({ code, message }: FileError): JSX.Element => {
+  /**  updating this specific error code because the language is awkward. Default code from react-dropzone is 'file-invalid-type'. The other error codes are fine. */
+  const errorCode = code === ErrorCode.FileInvalidType ? "incorrect-file-type" : code
+  const notificationTitle = prettifyErrorCode(errorCode)
+
+  return (
+    <InlineNotification
+      type="negative"
+      title={notificationTitle}
+      persistent
+      noBottomMargin={true}
+    >
+      <span
+        tabIndex={0}
```

Comment:
To ensure that all error messages are read by the screen reader, this `tabIndex` needs to be applied to your container. eg.
```
<div tabIndex={-1}>
    <FieldMessage />
    <FieldMessage />
</div>
```

Reply:
Also generally this `span` (and it's remaining props) won't be needed when converted to use `FieldMessage`